user_tools/src/spark_rapids_pytools/resources/dev/log4j.properties

```diff
@@ -0,0 +1,20 @@
+# Set root logger level to INFO and its only appender to file
+log4j.rootLogger=INFO, FILE, stderr
```

Comment:
is this documented on how to set logging somewhere?  Either user or dev docs or both.

Reply:
Currently it is not documented.  
For dev docs, can we add it here since it captures core tools log - https://github.com/NVIDIA/spark-rapids-tools/blob/dev/core/README.md ?